### DISCUSSION TITLE
Aidez moi svp

### DISCUSSION THREAD

**Discussion by IvaneTch** [PrestaSafe/prettyblocks#178] [Q&A]
![Capture d’écran 2023-12-25 à 17 23 13](https://github.com/PrestaSafe/prettyblocks/assets/112415607/ea9f4b75-e6c9-46f7-91f4-6f549836f699)

Bonjour
Aidez moi svp j'arrive pas utiliser "Pretty Blocks" ca me demande Api Key j'ai Key mais je ne sais pas quai faire.

**Comment by PrestaSafe** [comment]
Bonjour, ce message est lié à TinyMce, normalement ce dernier disparait au bout d'une demi seconce. Avez vous des erreurs dans la console javascript ? Cordialement

**Comment by IvaneTch** [comment]
Bonjour Merci pour votre réponse.Oui ce disparaît mais les blocks ne s’affichent pas et j’ai pensé que c’était à cause de ça.Pour vous expliquer, j’ai créé une nouvelle page Dans les pages. Après j’ouvre dans Pretty blocks et j’essaye de rajouter bannière. Il écrit que c’était bien rajouté mais je ne vois pas, que sur les pages index s’affiche quand je rajoute mais j’arrive pas ajouter une photo par exemple un texte, il s’affiche que avec le taille des photos  et le petit texte, mais quand j’apporte le modification ça change rien pouvez-vous s’il vous plaît m’aider j’ai besoin très urgent modifier quelques pages.Merci d’avance Cordialement ivane Le 26 déc. 2023 à 11:19, Guillaume Batier ***@***.***> a écrit :﻿
Bonjour, ce message est lié à TinyMce, normalement ce dernier disparait au bout d'une demi seconce. Avez vous des erreurs dans la console javascript ? Cordialement

**Comment by ghost** [comment]
Bonjour,
Le message disparait rapidement en effet mais le block reste en read-only. Sur le site de Tiny il est indiqué qu'à partir de 2024, si aucune API Key valide n'est renseignée, le module sera en Read-only.

Il faut donc une clé d'API valide. Est-ce à vous de la fournir ? A nous de s'inscrire ?
Merci

**Comment by PrestaSafe** [comment]
Bonjour, tout est mentionné ici: https://github.com/PrestaSafe/prettyblocks/wiki/Probl%C3%A8me-de-cl%C3%A9-API-du-TinyMCE-(FR)
